generators/chipyard/src/main/scala/TileOnlyConfig.scala

```diff
@@ -0,0 +1,210 @@
+package chipyard
+
+import chisel3._
+import org.chipsalliance.cde.config.{Parameters, Config, Field}
+import freechips.rocketchip.diplomacy._
+import freechips.rocketchip.interrupts._
+import freechips.rocketchip.tile._
+import freechips.rocketchip.subsystem.RocketCrossingParams
+import freechips.rocketchip.tilelink._
+
+case object RocketTileOnly extends Field[RocketTileParams]
+
+
+class TileOnlyTestHarness(implicit p: Parameters) extends Module {
+  val io = IO(new Bundle {
+    val success = Output(Bool())
+  })
+  println("Elaborating TileOnlyTestHarness")
+
+  io.success := false.B
+
+  val lazyDut = LazyModule(p(BuildSystem)(p)).suggestName("TileOnlyDigitalTop")
```

Comment:
```suggestion
  val lazyDut = LazyModule(p(BuildSystem)(p)).suggestName("TileOnlyDigitalTop").asInstanceOf[TileOnlyDigitalTop]
```

I believe if you do this, then you don't have to do the `dut match {...` below, and I don't think the loss of generality is an issue here.

Reply:
Thanks!